MAUI/ListView/viewappearance.md

```diff
@@ -0,0 +1,1208 @@
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Appearance in .NET MAUI ListView (SfListView)
+
+The `SfListView` allows customizing appearance of the underlying data, and provides different functionalities to the end-user.
+
+## Item template
+
+A template can be used to present the data in a way that makes sense for the application by using different controls. `SfListView` allows customizing appearance of view by setting the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate) property.
+
+## Data template selector
+
+The `SfListView` allows customizing appearance of each item with different templates based on specific constraints by using the `DataTemplateSelector`. You can choose a `DataTemplate` for each item at runtime based on the value of data-bound property using `DataTemplateSelector`.
+
+Here, an [ItemsCacheLimit](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ListViewLayout.html#Syncfusion_Maui_ListView_ListViewLayout_ItemsCacheLimit) property maintains number of items reusing in the view. This cache limit is used to create and reuse the [ListViewItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ListViewItem.html) if different templates are used in `DataTemplateSelector` for better scrolling performance. Based on this value, `SfListView` creates number of `ListViewItem` for different templates in the view if new template is created while scrolling, and reuses it if same template is used for improving the scrolling performance.
+
+### Create a data template selector
+
+Create custom class that inherits from `DataTemplateSelector`, and override the `OnSelectTemplate` method to return the `DataTemplate` for that item. At runtime, the SfListView invokes the `OnSelectTemplate` method for each item and passes the data object as parameter.
+
+{% tabs %}
+{% highlight c# %}
+
+class MyDataTemplateSelector : Xamarin.Forms.DataTemplateSelector
+{
+
+   private readonly DataTemplate incomingDataTemplate;
+   private readonly DataTemplate outgoingDataTemplate;
+
+   public MyDataTemplateSelector()
+   {
+      this.incomingDataTemplate = new DataTemplate(typeof(IncomingViewCell));
+      this.outgoingDataTemplate = new DataTemplate(typeof(OutgoingViewCell));      
+   }
+
+   protected override DataTemplate OnSelectTemplate(object item, BindableObject container)
+   {
+      var message = item as Message;
+      if (message == null)
+         return null;
+      return message.IsIncoming ? this.incomingDataTemplate : this.outgoingDataTemplate;
+   }
+
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Applying the data template selector
+
+Assign custom `DataTemplateSelector` to the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate) of the `SfListView` either in XAML or C#.
```

Comment:
Assign a custom `DataTemplateSelector` to the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate) of the `SfListView` in XAML or C#.

Reply:
I have modified the line as per the concerns provided.